src/main/java/io/codelex/flowcontrol/practice/PhoneKeyPad.java

```diff
@@ -0,0 +1,51 @@
+package io.codelex.flowcontrol.practice;
+
+import java.sql.SQLOutput;
+import java.util.Scanner;
+
+public class PhoneKeyPad {
+    public static void main(String[] args) {
+        Scanner in = new Scanner(System.in);
+        System.out.println("Please enter a string! ");
+        String input = in.next().toLowerCase();
+
+        letterGenerator(input);
+
+    }
+
+    private static void letterGenerator(String input) {
+        for (int i = 0; i < input.length(); i++) {
+            char letterDigit = input.charAt(i);
+            int digit = 0;
+
+            switch (letterDigit) {
```

Comment:
Šito pārtaisi par new style switch. Man liekas IDEA jau piedāvā to izdarīt - tā būs stipri īsāk.